### PR TITLE
First attempt

### PR DESCRIPTION
**Pull request by ggzsc**
This is my Community Contribution. Thanks a lot.

### CODE REVIEW

File: Tidyver_Help.Rmd [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+```{r}
+# we can omit '.'
+c(1, 3, 4, 5, NA) %>% mean(., na.rm=TRUE)
+c(1, 3, 4, 5, NA) %>% mean(na.rm = TRUE)  
+# we can not omit '.' in the second parameter
+c(1, 3, 4, 5) %>% plot(., main=paste(., collapse=", "))
+c(1, 3, 4, 5) %>% plot(main=paste(., collapse=", "))
+```
+
+### 0. Connection Bewtween two datasets
```
Comment (joywang1999) [inline Tidyver_Help.Rmd:20]: Please remove the numbers here and all below, use only multiple hashtags.
```suggestion
### Connection Bewtween two datasets

...

### Combine two datasets by value: join

...
```

**Review by joywang1999 — CHANGES_REQUESTED** [review]

**Comment by joywang1999** [comment]
Please change your filename to all lowercases (ex. `tidyver_help.Rmd`), thanks!

File: Tidyver_Help.Rmd (hunk 1 above)
Comment (ggzsc) [reply Tidyver_Help.Rmd]: I commit the second edition. I'm really sorry for missing this conversation.

Comment (joywang1999) [reply Tidyver_Help.Rmd]: I saw you fixed the hashtags! Could you also change your filename to all lowercases (ex. tidyver_help.Rmd)? Thanks!

**Comment by ggzsc** [comment]
Dear Jinyu Wang:
  Do I need to create a new file and submit? I rename it in r and commit it
last time but it seems that it doesn’t work. How should I fix it?

Sincerely,
Zhisen

Jinyu Wang ***@***.***>于2021年11月10日 周三下午7:07写道：

**Comment by joywang1999** [comment]
Hi Zhisen, I renamed you file. You are good to go. Just a reference on how to do it, simply open the file on github.com and click the edit bottom as shown below. 
image
You can now edit your file without dealing with git commands locally. Here I renamed the file by editing the file name, then clicked `Commit Changes` 
image

**Comment by joywang1999** [comment]
Thank you for your contribution! 🎉

**Comment by ggzsc** [comment]
Thank you for your detailed explanation!!! Have a nice day
Sincerely,
Zhisen

**Pull request merged**